plugin/adsk/plugin/exportTranslator.cpp

```diff
@@ -50,6 +50,9 @@ MStatus UsdMayaExportTranslator::writer(
     const MString&                    optionsString,
     MPxFileTranslator::FileAccessMode mode)
 {
+    MStatus s = MPxFileTranslator::writer(file, optionsString, mode);
```

Comment:
Don't get this...  You're calling the base class writer(), but the Maya docs say this fails unconditionally, and I just looked at the code, and it does fail unconditionally...  And you're checking for not failure, so this is all a no-op.  I must be missing something.